src/test/scala/uk/gov/hmrc/perftests/debttransformation/requests/InterestForecastingRequests.scala

```diff
@@ -23,12 +23,42 @@ import uk.gov.hmrc.performance.conf.ServicesConfiguration
 
 object InterestForecastingRequests extends ServicesConfiguration {
 
+//  Assumptions
+//  1 Debt item
+//  Initial amount and date
+//    REGIME == DRIERd
+//  Where charge type is DRIER NI, then interest bearing is assumed to be true
+//  Where charge type is DRIER HiPG, then interest bearing is assumed to be false
+//  NO repayments
+//    NO suppression period
+//  NO breathing space
+//  Date Amount  == Interest start date
+//    No outstanding interests to pay
+//  When bearing the interest rate is 1%
+//
+//  DTD-191: IFS Amounts to be in pennies. Is outstanding
+
+
   val bearerToken    = BaseRequests.creatAuthorizationBearerToken(enrolments = Seq("read:interest-forecasting"))
-  val requestHeaders = Map("Authorization" -> s"Bearer $bearerToken", "Accept" -> "application/vnd.hmrc.1.0+json")
+  val requestHeaders = Map("Authorization" -> s"Bearer $bearerToken", "Accept" -> "application/vnd.hmrc.1.0+json","Content-Type"-> "application/json")
 
   def InterestForecastingRequestsHelloWorld(baseUri: String): HttpRequestBuilder =
     http("GET Interest Forecasting")
       .get(s"$baseUri/interest-forecasting/hello-world")
       .headers(requestHeaders)
       .check(status.is(200))
+
+  def InterestBearingdrierdebtForChargeTypeNino(baseUri: String): HttpRequestBuilder =
```

Comment:
Should this be InterestBearingdrierdebtForChargeTypeNI?

Reply:
I guess it can be but i thought I use "nino" which is widely used even in production code

Reply:
Thanks for explaining